### ISSUE TITLE
article-meta outputs escaped html so browser can't render it

### ISSUE THREAD

**Issue by solomkinmv** [canstand/compost#2]
Hello, thank you for your theme. I really like how minimal and light it is!
I've noticed strange behavior when [article-meta](https://github.com/canstand/compost/blob/main/layouts/partials/article-meta.html) concats all meta partials it outputs following html:

```html
&lt;time datetime=&#34;2019-03-11 00:00:00 &amp;#43;0000 UTC&#34;&gt;11 March 2019&lt;/time&gt;&lt;span class=&#34;px-2 text-primary-500&#34;&gt;&amp;middot;&lt;/span&gt;&lt;span title=&#34;Reading time&#34;&gt;3 mins&lt;/span&gt;&lt;span class=&#34;px-2 text-primary-500&#34;&gt;&amp;middot;&lt;/span&gt;
```

<img width="819" alt="image" src="https://github.com/canstand/compost/assets/9872823/be4a7928-18b7-4a5b-8c6b-cf1b92ee64a2">

The same happens for my site and for your example site.

I'm using go version `go1.21.5 darwin/arm64`.
I've tried theme versions `v0.6.0` and `v0.6.3`.

Removing concatenation and appending partials manually fixes the problem.

```html
    {{ with ($meta.Get "partials") }}
      {{ delimit . "<span class=\"px-2 text-primary-500\">&middot;</span>" }}
    {{ end }}
```
to
```html
    <time datetime="{{ .Date }}">
      {{- .Date.Format .Site.Params.article.dateFormat | default "2 January 2006" -}}
    </time>
    <span class="px-2 text-primary-500">&middot;</span>
    <span title="{{ i18n "article.reading_time_title" }}">
    {{- i18n "article.reading_time" .ReadingTime | emojify -}}
    </span>
```

Manual usage of all three partials also works fine.

I've tried to find the problem with the template, but I couldn't.

**Comment by canstand** [comment]
fixed with [v0.6.4](https://github.com/canstand/compost/releases/tag/v0.6.4)

**Comment by solomkinmv** [comment]
Thanks, now it works!
